Tests/test_SeqUtils.py

```diff
@@ -266,6 +266,9 @@ def test_codon_adaptation_index_initialization(self):
         )
         value = cai.calculate("ATGCGTATCGATCGCGATACGATTAGGCGGATG")
         self.assertAlmostEqual(value, 0.70246, places=5)
+        optimized_sequence = cai.optimize("ATGCGTATCGATCGCGATACGATTAGGCGGATG")
+        optimized_value = cai.calculate(optimized_sequence)
+        self.assertEqual(optimized_value, 1.0)
```

Comment:
Can you add a test where the optimiszed value is not one?

Reply:
Thanks for feedback! The value for an optimized sequence should always be 1. I could test that the optimized sequence still codes for the same amino acids as the original sequence, though. I could also do a test using amino acid sequence as the input; I have used that option less in practice.

Reply:
You can see I've not looked at the caculation details ;)